src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -241,6 +242,29 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
       .length,
   )
 
+  const isOpportunityMature = useMemo(() => {
+    if (!isRunePool) return true
+    if (!isRunePoolOpportunity(earnOpportunityData)) return false
+    if (!earnOpportunityData?.maturity) return false
+
+    const maturityDate = new Date(earnOpportunityData.maturity)
+    const currentDate = new Date()
+
+    return currentDate >= maturityDate
```

Comment:
q: feel free to close me since I'm always a derp re: time things, but will this handle timezones/offsets properly? i.e do we want to use `dayjs` here instead?

Reply:
Also another q, possibly paranoia: 

While the checks make absolute sense, do we want to add some sanity check re: `thorchainBlockTimeMs * RUNEPOOL_MINIMUM_WITHDRAW_BLOCKS`?
While `thorchainBlockTimeMs` should be *very* close to 100% accuracy WRT the actual blocktime, it may not be 100% accurate, and we may want to double check that the maturity indeed has withdraw unlocked with a dummy withdraw quote?

Reply:
> Also another q, possibly paranoia:
> 
> While the checks make absolute sense, do we want to add some sanity check re: `thorchainBlockTimeMs * RUNEPOOL_MINIMUM_WITHDRAW_BLOCKS`? While `thorchainBlockTimeMs` should be _very_ close to 100% accuracy WRT the actual blocktime, it may not be 100% accurate, and we may want to double check that the maturity indeed has withdraw unlocked with a dummy withdraw quote?

How would you check the maturity with a dummy withdraw quote? I don't get it

Reply:
> q: feel free to close me since I'm always a derp re: time things, but will this handle timezones/offsets properly? i.e do we want to use `dayjs` here instead?

As we don't manipulate the date so much, dayjs feel useless

It should handle timezone properly as thorchain provide us a timezone in UTC format (precised by the Z ad the end ie `2024-08-02T10:48:07.872822075Z`), on my computer this date indicate `12:48` as it should